aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/util/ImmutabilityHelper.java

```diff
@@ -0,0 +1,88 @@
+package software.amazon.rds.dbinstance.util;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+
+import com.google.common.base.Objects;
+import software.amazon.rds.dbinstance.ResourceModel;
+
+public final class ImmutabilityHelper {
+
```

Comment:
Should general immutable parameters be removed from Translator.modifyDbInstanceRequest as they will be never executed and they can be misleading for anyone reading code? A comment there referencing this class can be helpful.

Reply:
Done